javascript-modules-engine/tests/jahia-module/src/react/server/views/testReactVersion/TestReactVersion.jsx

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React from "react";
```

Comment:
```suggestion
import React from 'react';
```

Reply:
This will no longer be an issue at the end of today!